src/stories/SegmentedPicker/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState } from "react";
+import clsx from "clsx";
+import Text from "../Typography/Text";
+import { SegmentedPickerProps, Segment } from "./type";
+
+export const SegmentedPicker = <T,>({
+  className,
+  segments,
+  value,
+  onSelect,
+}: SegmentedPickerProps<T>) => {
+  const [selectedSegment, setSelectedSegment] = useState<Segment<T> | null>(
+    segments.find((segment) => segment.value === value) || null
```

Comment:
if not provide value, set default as segments first item value

Reply:
```
const [selectedSegment, setSelectedSegment] = useState<T>(
  segments.find((segment) => segment.value === value)?.value ?? segments[0].value
 );
 ```

Reply:
change:
  const [selectedSegment, setSelectedSegment] = useState<T>(
    segments.find((segment) => segment.value === value)?.value ||
      segments[0].value
  );